File: grad/Summer.java
public class Summer {

    public static void main(String[] args) {
        int sum = 0;



        for(int i = 1; i <= 100000 ; i = i + 1) {
            if(i % 2 == 0   ){
                sum = sum + i;
            }

        }

        System.out.println(sum);

    }
}

File: itp/summer.py

print("Please enter some numbers. I will sum them up.")
print("Enter \"done\" when you are done")
userInput =  input()
total = 0

while userInput != "done":
    number =  int(userInput)
    total += number
    userInput =  input()
    userInput = userInput.lower()

print(total)

    
"""
number = int(input("\n"))
while number > 0: 
    print("that was a positive number")
    number = int(input("I'd like a negative number, actually.\n"))
    print("Thanks, I'm gonna check your number")
    
print("done")
"""
